sdk/test-utils/recorder/tsconfig.browser.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./.tshy/build.json",
+  "include": ["./src/**/*.ts", "./src/**/*.mts", "./test/**/*.spec.ts"],
+  "exclude": ["./test/**/node/**/*.ts"],
+  "compilerOptions": {
+    "outDir": "./dist-test/browser",
+    "rootDir": ".",
+    "skipLibCheck": true
```

Comment:
Talk to me friendo, why do we need to skip lib check?

Reply:
Until we fix the whole browser testing stuff, it's just this way for now.